backend/src/main/resources/db/migration/V1_0_11__Update_Information_About_Profile.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE credentials
+    ADD COLUMN phone_number NUMERIC(9,0) NULL;
+ALTER TABLE credentials
+    ADD COLUMN birth_date DATE NULL;
+
+ALTER TABLE locations
+    ADD COLUMN zip_code CHAR(5) NOT NULL;
+ALTER TABLE locations
+    ADD COLUMN island VARCHAR(15) NOT NULL;
+ALTER TABLE locations
+    ALTER COLUMN province DROP NOT NULL;
+ALTER TABLE locations
+    ALTER COLUMN town DROP NOT NULL;
+ALTER TABLE locations
+    ALTER COLUMN address DROP NOT NULL;
+
+ALTER TABLE volunteers
+    DROP COLUMN age;
+
+ALTER TABLE volunteers
+    ADD COLUMN twitter VARCHAR(255) NULL;
+ALTER TABLE volunteers
+    ADD COLUMN instagram VARCHAR(255) NULL;
+ALTER TABLE volunteers
+    ADD COLUMN linkedin VARCHAR(255) NULL;
+ALTER TABLE volunteers
+    ADD COLUMN photo VARCHAR(255);
```

Comment:
Is 255 enough?

Reply:
Hi, Ibai, I find here the information about the length of the data in storage in A3:

https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html

"The name for a key is a sequence of Unicode characters whose UTF-8 encoding is at most 1,024 bytes long."

Might we to change the lenght in database to 1024?

Reply:
It's done. I have changed the length to 1030 characters (6 more over the limit).